### DISCUSSION TITLE
Available outside of chrome://new-tab

### DISCUSSION THREAD

**Discussion by wimbarelds** [dailydotdev/daily#1589] [Feature Requests]
I use a "speed-dial" new tab extension (specifically 'yet another speed dial'), and I like using it.

But I would like to be able to use daily.dev. It would be great if I could access it some way other than new-tab. Perhaps there is some very obvious way I can do this, but I cannot find it.

[edit] I found the small "web version" link. I guess that makes this request moot, except maybe make that option more prominent on the website?

**Comment by nimrodkra** [comment]
You can use daily.dev as a web app via https://app.daily.dev and either
save it as a bookmark/shortcut or configure it as homepage (or just use it
as a normal website).
